Test/Pester3.Tests.ps1

```diff
@@ -349,14 +349,14 @@ Describe 'PassingTests' {
     }
 }
 
-Describe 'Pester3.when a task path is absolute' {
+Describe 'Pester3.when a task path is outside the build root' {
     AfterEach { Reset }
     It 'should fail' {
         Init
         GivenTestContext
-        GivenTestFile 'C:\FubarSnafu'
+        GivenTestFile '..\FubarSnafu.ps1'
         WhenPesterTaskIsInvoked -ErrorAction SilentlyContinue
         ThenPesterShouldHaveRun -PassingCount 0 -FailureCount 0
-        ThenTestShouldFail -failureMessage 'absolute'
+        ThenTestShouldFail -failureMessage 'outside\ the\ build\ root'

```

Comment:
Seems odd to escape spaces.

Reply:
I do it because that's what `[regex]::Escape` does:

```
# [regex]::Escape('    ')
\ \ \ \ 
```